### ISSUE TITLE
lodash uniqWith doesn't work

### ISSUE THREAD

**Issue by FinnStutzenstein** [OpenSlides/OpenSlides#2293]
With Lodash version 4.0.0 the `_.uniqWith` feature is introduced. At the moment, you have to upgrade to version 4.0.0 manually. It would simplify the code in the `$scope.import` function (in motions `side.js`; there is a TODO mark, PR:  #2286) a lot, but this feature doesn't work.

The [test](https://lodash.com/docs#uniqWith) from lodash.com went wrong:
```
var objects = [{ 'x': 1, 'y': 2 }, { 'x': 2, 'y': 1 }, { 'x': 1, 'y': 2 }];
_.uniqWith(objects, _.isEqual);
// expected result: [{ 'x': 1, 'y': 2 }, { 'x': 2, 'y': 1 }]
```
But it does not filter duplicates; the result is equal to `objects`.

If it is working, we could think about update to lodash version 4 generally, because there are some new helpful features.

**Comment by FinnStutzenstein** [comment]
Since using pdfmake, I can not change the lodash version, because lodash is already included in pdfmake with version 3.10.1. The result are two different versions of lodash (3.10.1 and 4.15.0) in the `openslides-libs.js`. If i print `_.VERSION` I get the 3.10.1 version.

How do we solve this problem?

**Comment by FinnStutzenstein** [comment]
There is an open issue for this problem: https://github.com/bpampuch/pdfmake/issues/276

**Comment by normanjaeckel** [comment]
You can find out or ask when the fix in pdfmake will be released. Then we can decide whether to wait or to make a temporary fork.

**Comment by FinnStutzenstein** [comment]
There is a fix with PR #443 in the newest version 0.1.20, but it doesn't work for me. It takes about a year to merge this PR and the latest updates were on may, so I don't think it is maintained very well.
The easiest solution ist in my perspective a fork, because we only have to remove lodash from the dependencies. And we could still use version 0.1.17 and don't have to upgrade.
